README.md

```diff
@@ -2,3 +2,11 @@
 **DISCLAIMER:** Robin is a repository designed to be a teaching tool for entry-level developers to better understand implementing features.  It's not meant to be a real product.
 
 `Robin` allows users to manage posts on all of their social media networks. 
+
+### Setting up and running the local Go server
+1) Install Go from [the official Go website](https://go.dev/)
+2) From a terminal, navigate to the `/backend` directory
```

Comment:
Rather than `backend`, let's call it server.  `backend` may also imply something like ORMs, database-specific stuff, etc.  `server` is pretty specifically talking about the web server.

Reply:
Fixed in latest push